build/gulp/tasks/bundle.ts

```diff
@@ -88,10 +94,14 @@ task('build:dist:umd', cb => {
   })
 })
 
-task('build:dist', parallel('build:dist:commonjs', 'build:dist:es', 'build:dist:umd'))
-
 // ----------------------------------------
 // Default
 // ----------------------------------------
 
-task('dist', series('clean:dist', 'build:dist'))
+task(
+  'bundle:package:no-umd',
+  series('bundle:package:clean', parallel('bundle:package:commonjs', 'bundle:package:es')),
+)
+task('bundle:package', series('bundle:package:no-umd', 'bundle:package:umd'))
+
+task('bundle:all-packages', () => sh('lerna run build'))
```

Comment:
ok, lets compromise on that for now :) in either case, this is quite easy thing to change _in case if needed_. Thank you!